ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -246,8 +246,9 @@ public void reset() {
         Collection<GlobalVariableSymbol> globalVariableSymbols = globalVariableSymbols(project, compilation);
         Map<QuotedIdentifier, GlobalVariable> allNewVariables = new HashMap<>();
         for (VariableDeclarationSnippet snippet : variableDeclarations.keySet()) {
+            String type = snippet.getRootNode().typedBindingPattern().typeDescriptor().kind().name();
```

Comment:
No need to use its name. Can just directly use the kind enum.

Reply:
Further, this can just be a simple boolean value. We are only interested in knowing whether the var was declared using `var` or not.

Reply:
Updated in 3fc03c0dee2d7091929456e76b603ab332cb786e